spec/features/parental_consent_flu_spec.rb

```diff
@@ -0,0 +1,113 @@
+require "rails_helper"
+
+RSpec.describe "Parental consent" do
+  scenario "Flu campaign" do
+    given_a_flu_campaign_is_underway
+    when_i_go_to_the_consent_form
+    then_i_see_the_consent_form
+
+    when_i_give_consent
+    then_i_see_the_first_health_question
+
+    when_i_answer_the_health_questions
```

Comment:
```suggestion
    when_i_answer_no_to_all_health_questions
```
or `when_i_dont_provide_any_notes_to_any_health_questions` or similar... (this wasn't obvious until I read the implementation of the step)